src/sentry/static/sentry/app/stores/groupingStore.jsx

```diff
@@ -214,60 +196,54 @@ const GroupingStore = Reflux.createStore({
   },
 
   // Toggle unmerge check box
-  onToggleUnmerge(id) {
+  onToggleUnmerge([fingerprint, eventId]) {
     let checked;
 
     // Uncheck an item to unmerge
-    let state = this.unmergeState.has(id) && this.unmergeState.get(id);
+    let state = this.unmergeState.has(fingerprint) && this.unmergeState.get(fingerprint);
```

Comment:
why not just `let state = this.unmergeState.get(fingerprint);`?

Reply:
Habit from mobx observables, it's unnecessary here.